scripts/copy_oppia_assets.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# INSTRUCTIONS:
+# This script copies all oppia-assets to domain/src/main/assets folder in
+# oppia-android project and also makes sure that they are not getting committed to github.
+#
+# Run the script from the oppia-android root folder:
+#
+#   bash scripts/copy_oppia_assets.sh <local_path_to_oppia_assets>
+#
+# NOTE: Once this script is run, we can build a signed APK and after that run scripts/copy_dummy_assets.sh
+
+source_assets_path="$1"
+destination_assets_path="domain/src/main/assets"
+
+# Copy current assets files outside local repo
+rsync -r "$destination_assets_path/" ../old-assets
+
+# Remove current assets from domain/src/main/assets
+rm -r domain/src/main/assets/*
+
+# Add .gitignore for the assets folder
```

Comment:
I think this comment belongs next to the next ``if`` block since it seems like all this one is doing is ensuring the path has a trailing ``/``.

Reply:
Yeah, that is correct. Updated comment accordingly.